File: Arcade/Intro/Java/AlmostIncreasingSequence.java
final class AlmostIncreasingSequence {

	boolean almostIncreasingSequence(int[] sequence) {
	    int last = -0x186a0, lastPrev = -0x186a0, c = 0;
	    
	    for(int n : sequence) {
	        if(n <= last) {
	            c++;
	            if(n > lastPrev) last = n;
	            continue;
	        }
	        lastPrev = last;
	        last = n;
	    }
	    
	    return c <= 1;
	}
}

File: Arcade/Intro/javascript/almostIncreasingSequence.js
function almostIncreasingSequence(sequence) {
    let l = -0x186a0, lp = -0x186a0, c = 0;
    
    for(let e of sequence) {
        if(e <= l) {
            c++;
            if(e > lp) l = e;
            continue;
        }
        lp = l;
        l = e;
    }
    
    return c <= 1;
}